packages/lesswrong/components/collections/CollectionsCard.jsx

```diff
@@ -17,31 +17,39 @@ const styles = theme => ({
   }
 })
 
-const CollectionsCard = ({collection, big = false, url, classes}) => {
-  const cardContentStyle = {borderTopColor: collection.color}
+class CollectionsCard extends PureComponent {
+  handleClick = (event) => {
```

Comment:
I'm not familiar with this syntax. Is "handleClick" a global variable, or is it a class-level variable? Either way, the recommended way of doing this (from the react docs) is:

```
constructor(props) {
  super(props);
  this.handleClick = this.handleClick.bind(this);
}

handleClick() {
   ...
}
```

See https://reactjs.org/docs/react-component.html#constructor